conf/routes

```diff
@@ -3,20 +3,20 @@ GET     /health                             controllers.HomeController.healthChe
 
 GET     /hello                              controllers.HomeController.hello(name:String)
 GET     /google                             controllers.HomeController.google
-GET     /authenticate/:provider             controllers.SocialAuthController.authenticate(provider, redirect:Option[String], postAuthRedirect: Option[String])
 
-GET     /sign-out                           controllers.HomeController.signOut
```

Comment:
the `/api` has to do with how the load balancer works. I'm routing `/api*` to the backend and everything else to the front end.